pyqtgraph/flowchart/library/Display.py

```diff
@@ -11,24 +14,24 @@ class PlotWidgetNode(Node):
     """Connection to PlotWidget. Will plot arrays, metaarrays, and display event lists."""
     nodeName = 'PlotWidget'
     sigPlotChanged = QtCore.Signal(object)
-    
-    def __init__(self, name):
+
+    def __init__(self, name: str) -> None:
         Node.__init__(self, name, terminals={'In': {'io': 'in', 'multi': True}})
-        self.plot = None  # currently selected plot 
-        self.plots = {}   # list of available plots user may select from
-        self.ui = None 
-        self.items = {}
-        
-    def disconnected(self, localTerm, remoteTerm):
+        self.plot: Optional[PlotWidget] = None  # currently selected plot
```

Comment:
Instance properties should be typed at the class level, not in the `__init__` function (see [here](https://mypy.readthedocs.io/en/stable/class_basics.html#instance-and-class-attributes))... like this:

```python
class PlotWidgetNode(Node):
    items: dict[int, QGraphicsItem | PlotDataItem
    plot: PlotWidget | PlotItem | None  # personal preference, but I only use `Optional` for function args
    plots: dict[str, PlotWidget | PlotItem]  
    ui: ComboBox | None
    
    def __init__(self, name: str) -> None:
        Node.__init__(self, name, terminals={'In': {'io': 'in', 'multi': True}})
        self.plot = None  # <-- no need for typing here! Type is defined at class level.
```